TTS/bin/compute_embeddings.py

```diff
@@ -1,73 +1,45 @@
 import argparse
```

Comment:
In general, it is better to use the speaker_manager in this script to reduce the maintenance cost. 

Your changes here demand changes in the speaker_manager any ways.

Reply:
update it :)

Reply:
I updated :)